client.cpp

```diff
@@ -225,8 +230,7 @@ voltdb::Procedure *get_procedure(voltclient_object *obj, const char *name, int p
     return proc;
 }
 
-voltdb::Procedure *prepare_to_invoke(INTERNAL_FUNCTION_PARAMETERS, voltclient_object *obj)
-{
+voltdb::Procedure *prepare_to_invoke(INTERNAL_FUNCTION_PARAMETERS, voltclient_object *obj) {
```

Comment:
Nitpick: It'll be nice if new code can follow the coding style of the existing code instead of changing some of the existing code to follow a different style. Now the code has inconsistent coding styles.